src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-29">
+      <div class="calendar__cell calendar__cell--day1 "></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
Давай обмежимось лише класом calendar__cell і будемо використовувати :nth-child